routing/cross_mwm_connector.hpp

```diff
@@ -13,25 +14,153 @@
 
 namespace routing
 {
+// @todo(bykoianko) This using should be moved to a special file with TransitId class.
+// Identifier to find and a border edge in neighbouring mwm while cross mwm transition
+// for pedestrian, bicycle and car routing.
+using OsmId = uint64_t;
+
+namespace connector
+{
+uint32_t constexpr kFakeId = std::numeric_limits<uint32_t>::max();
```

Comment:
Is it a fake OsmId? Or fake feature id?

Reply:
It's fake (or even invalid) |m_enterIdx| and |m_exitIdx|. Please have a look at CrossMwmConnector::AddTransition() to see how it is used.